.circleci/config.yml

```diff
@@ -259,6 +259,7 @@ jobs:
       name: vm-<< parameters.arch >>
     parallelism: << parameters.parallelism >>
     steps:
+      - checkout
```

Comment:
checkout takes only 3 seconds so I think it's ok to trade this for a more reusable code

<img width="1441" alt="image" src="https://user-images.githubusercontent.com/2753650/183101001-5c266397-8c85-4678-a186-59505478b65d.png">